### PR TITLE
PPM weight slider resets if PPM size is changed.

### PR DESCRIPTION
**Pull request by Igarfinkle**
## Description

When the PPM slider is encountered for the first time, the slider is automatically set to the center of the range. If the service member returns to the previous page and chooses a new PPM size, the weight slider should reset to the center of the new range. This delivers that functionality. 

## Setup

Set up a new PPM move
Choose a weight class (S/M/L) and view the weight slider. 
Return to the size page and choose a new size. 
The slider resets to the center of the range.

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/165136769) for this change

### CODE REVIEW

**Comment by kahlouie** [comment]
@Igarfinkle love that you are tackling this, but this doesn't solve the bug of setting the weight estimate prematurely. The scenario that is still not addressed is:
1) SM chooses a size and goes to the weight slider
2) SM leaves the app and returns with a new session
3) SM continues move and is directed to the review part (problem being that the weight estimate has been set to the middle of the original size selected (and not given a chance to correct this mistake unless they catch it on review).

**Comment by kahlouie** [comment]
@sarboc @jim Isaac has asked me if there is a story for the things I have mentioned. I was wondering if either of you know.

**Comment by Igarfinkle** [comment]
We're wrapping the issues Kara addressed into this story and they should be fixed.

File: src/scenes/MyMove/getWorkflowRoutes.jsx [hunk 1]
```diff
@@ -231,7 +231,8 @@ const pages = {
   },
   '/moves/:moveId/ppm-incentive': {
     isInFlow: hasPPM,
-    isComplete: ({ sm, orders, move, ppm }) => get(ppm, 'weight_estimate', null),
+    isComplete: ({ sm, orders, move, ppm }) =>
+      get(ppm, 'weight_estimate', null) && get(ppm, 'weight_estimate', null) !== 0,
```
Comment (kahlouie) [inline src/scenes/MyMove/getWorkflowRoutes.jsx]: I think the null in the second get is unnecessary, but also not a blocker to shipping this.

File: swagger/internal.yaml [hunk 2]
```diff
@@ -202,7 +202,7 @@ definitions:
         x-nullable: true
       weight_estimate:
         type: integer
-        minimum: 1
+        minimum: 0
```
Comment (kahlouie) [inline swagger/internal.yaml]: Do we actually want to change the minimum on the api schema?

Comment (kahlouie) [reply swagger/internal.yaml]: I'm unsure of the ramifications of having a person make a move that is 0 lbs. In reality, 1 doesn't seem like a realistic sized ppm shipment either.

Comment (Igarfinkle) [reply swagger/internal.yaml]: After the SM chooses the weight class (S/M/L), we set the weight to 0 before the SM proceeds to the weight selector page. This is how we signify "The SM has completed the Size page but has not completed the Weight page" so if they leave their session, we return them to the right place.

**Pull request merged**